CHANGELOG.md

```diff
@@ -4,6 +4,44 @@
 
 - [app-functions-sdk-go](https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/CHANGELOG.md)
 
+## [v3.1.0] Napa - 2023-11-15 (Only compatible with the 3.x releases)
+
+
+### ✨  Features
+
+- Remove snap packaging ([#238](https://github.com/edgexfoundry/app-rfid-llrp-inventory/issues/238)) ([ae0fc91…](https://github.com/edgexfoundry/app-rfid-llrp-inventory/commit/ae0fc916ce7d511bfada48b200efa99a99f94c46))
+```text
+
+BREAKING CHANGE: Remove snap packaging ([#238](https://github.com/edgexfoundry/app-rfid-llrp-inventory/issues/238))
+
+```
+
+
+### ◀️ Revert
+
+- Restore ignored items that were not snap-related ([#241](https://github.com/edgexfoundry/app-rfid-llrp-inventory/issues/241)) ([df6d7f6…](https://github.com/edgexfoundry/app-rfid-llrp-inventory/commit/df6d7f608fd9c1c4fe6c937888fc634314559cbf))
```

Comment:
This was a Napa WIP issue, so it can be removed.

Reply:
ok. wasn't sure about this one.